src/services/facilityService.ts

```diff
@@ -1,13 +1,48 @@
-import { facilities } from '../mockData/mockData';
 import { Facility } from '../typeDefs/gqlTypes';
+import {Facility as PrismaFacility, Contact as PrismaContact } from '@prisma/client';
+
+import prisma from '../db/client';
+
+type FacilityAndRelations = (PrismaFacility & {
+    contact: PrismaContact;
+})
+
+function convertPrismaToGraphQLFacility(input: FacilityAndRelations) {
+    return {
+        id: String(input.id),
+        nameJa: input.nameJa,
+        nameEn: input.nameEn,
+        contact: input.contact,
+        healthcareProfessionals: []
```

Comment:
hydrating these objects is a bit complex. I'd love to work more on healthcareProfessionals in a following PR